dev/Omobolaji/vehicles.ipynb

```diff
@@ -0,0 +1,958 @@
nFpGV3cINmpNvVDdIdgwHjhU3SFUynj2YHWHYCPEw7+6Q7BhPLWnukOwIc+fqe4QbMjWrtUdQqVkbkZ1h2BDxp2s7hBseNTAe+mCfJfqDsGGa6quukOoVMCtztUdgg1NsF91h6DcwFTlVlEURVEURVEU5UanWm7VmFtFURRFURRFURTlxqdabhVFURRFURRFUW50UlZ3BNVOtdwqiqIoiqIoiqIoNzxVuVUURVEURVEURVFueKpbsqIoiqIoiqIoyo1OTSilWm4VRVEURVEURVGUG59quVUURVEURVEURbnRqZZb1XKrKIqiKIqiKIqi3PhUy62iKIqiKIqiKMqNTqqWW1W5VW5q249fYP6v/2CURoZ2asaYXm1s8qw9EMcaNlUyAAAgAElEQVTi9ftAQNPaAcx7qBcA7/2xi61HzwPweJ+2DIhq7JiYTiczf91BjFIyNKohY7o2s43pyAUWbz0KQNMQX+YN6cSx5GzmrNlPfokOrUbwn27NGdAizDExxcYxf/lGjEYjQ7u3YczAW63SFyzfSMzxeACKS3Vk5hWy7b2JxBw/x4Llf5blO5ucwbzH7qZ3VFOHxHUpM+a8w5btuwjw9+OXrz/+1/d30faTicxftdt0/NpHMOa2SJs8aw+dY/Gmg4Cgaagf84Z1JyYumQWr95blOZuew7z7u9O7RT27Y9I2icLlzkdBo0G/eyO6Lb9Uni/yVtwemkzRohcwJsQh/IJwn/gexvREAIznT1D66yd2xwOw/chZ5v/0F0ajZGiXSMb072iVvuCnv4g5eQGA4lI9mfmFbJs/viw9v6iEoW98Re/W4Uwb1sshMQG8Mmcq0X27U1RUzPNPv8rhg8es0j29PPjut8/K3ofWCebXH1bz+oy3qBNWmzc/eJWAQH+ys3OYPG4GyUmpdse0/Vg883/521RWnZszpk9bq/QFv/5NzCnTMTKVVRHb3ngUgCeXrOLguVTaNgrlw//cbncsZTGdTWfBX8cwGiVDWoYxpmMjq/SVhxN4d9sJgj3dAHggqh73tAwj5nwmb/11vCzf2awC5t3eml4RwQ6LDWD7oVO8+e1a07m9R1vG3tndKj0pI4cZn/1CXmEJRqORZ+/rQ4/WTRwaA8D2o+eY//M2Uxy3tmBM3/ZW6QtWbCv/nOv0ZOYVsW3eYyRm5jL58zUYjEb0RiPDe7Tm/m4tHRaXpkEkLj2Hmc4JsdvQ715rla5t0QWX7vciC7IB0O3fhOHw9vIMLm64PTwTw6n96DZ/57C4Os4eRd3eURiKStg+aQmZsWdt8kS9cD/h93XHxdeTZU3/U7bcs04g3d5/AhcfD4RGw96535Pw54GrjsGrZzvqvvIYaDVkfr+etI9+tEoXLk7Ue+c53FuGY8jO49xT89FdSMU5LJhmGxZREpcAQOG+4yRMXwRAyJRR+N/TC62vF4cjh111TJZcb+2I33NPITQaClb+Qd7/llmlu0S1xm/SBJwjGpP58msU/bnFtF77KHwnPlmWz7lBfTJmvEbxlu3YS9uiPW73jwOhQff3GkrX/WCV7tzjDpxvuwuMRmRJMSXffoAxOR40WtxGTkRTLxy0WvQ7N1K6drnd8ZTFFdEGl4EPmz7nezeh27ayivg74TZsEkVLpmNMjEPbqhvO3e4qS9eE1Kd48UsYk885LDal+qjKrXLTMhiNzF3xNx8/NpAQX09GfLiSni3qEx7iX5bnXFoOn286wJdP3oWPhyuZ+UUAbDkaz9GEDL6fOBSdwcDYj/6gW/MwvNxc7IxJMnfNAT5+qDshPu6M+HwTPZvUJjzIpzymzHw+//s4Xz7cEx93FzILigFwd9by2t0daBDgRWpeEQ999iddGgfjY3dMRuYuW8/HEx8gxN+bEXOX0rN1BOF1apXlmTqsT9nrZX/u4dj5FAA6NmvA8pdNN9k5BUUMmrGELi2sb4L/LUPu6MdD997NS6+9dV32B+ay+i2Gj0f3JsTHgxEfr6Fn8zDCg33L8pzLyOXzLYf58rH++Li7kplvOn4dG4eyfMIdAOQUljDovZV0iahtf1BCg8ugsRR/8RoyNxO38XPRH92NTLtgnc/FDecut2OIP2G1WGYmU/zfqfbHYcFgNDL3h818PGEoIX5ejFjwHT1bNSa8dmBZnqn39ix7veyv/Ry7kGa1jYWr/qF9RF2HxhXdtxsNG9end6fBRLVvxewF07h3wCNWeQryCxnUa3jZ+183fsPaVaYHONNmTWTF97/z8/e/06VHR6a8/DRTnnzZrpgMRiNzf97Ox0/caTpPvfczPSMbEh5afp6aOrhr2etlW2M5lpBe9v6R6DYU6/T8uOOoXXFYxySZt+koH93TnhAvN0Ys+4eejYMID/SyyjegaSgv9rrFalnHegF8P7ILADnFOu7+Yiu3NgjEkQxGI3O+Xs3iySMJCfDhodmfEh3VjPC6QWV5PvltKwM6RjKsVwdOJ6Tx1HvfsnrBsw6PY+6PW/h4/N2mz/k7P9CzZSPCQwPK8kwdWl7pXrblYNnnPMjHk6UT78XFSUthSSn3zvuOni0bEezraX9gQuDSazglP7+HzM/Cbfg0DHEHkZlJVtn0J3ZXWXF17nI3xgsn7Y/FQt3ebfBpFMov3SdTq104neeOZvWgmTb5Lqzfy/Ev1jNkm/W5vdWzgzn7205O/G8jvk3q0Oerqfx866SrC0Kjoe7scZwZ+TK65AwiVr5D7vqdlJw6X5YlYFh/DDn5HI9+At9BPaj94mjin5oPQOm5ZE7eYfs5ytu4i4ylv9Ns8+Kri6eS+PynPkva01MxpKYR/OVHFG39G/2Z8kqXISWFrNfexGuEdSW6ZM9+Ukc9DoDw8ab2j19RsnO3ffEACA1uD0yg8IOXkNnpeLzwPvqDO02VVzNdzGZ0W/8AQNuqM673PkbRwpdxatcDnJwpfONJcHbF85XF6GI2IzPtfyiIELjc8SjFX81B5mbg9tgb6I/vQaYlWOdzccO580AMFp9nw6HtGA6Z
```

Comment:
Also, please add a comment describing how you decided on this train-test split ratio. We are asking for this input at this point because one of goals of the project is to better understand how this choice influences the outcomes of the model.

Reply:
Okay, I will. Thanks